client/packages/invoices/src/Returns/InboundDetailView/Footer/StatusChangeButton.tsx

```diff
@@ -86,27 +123,29 @@ const getButtonLabel =
   };
 
 const useStatusChangeButton = () => {
-  // const { lines, status, onHold, update } = useOutbound.document.fields([
-  //   'status',
-  //   'onHold',
-  //   'lines',
-  // ]);
-
   const { success, error } = useNotification();
   const t = useTranslation('distribution');
-  const { data } = useReturns.document.inboundReturn();
-
-  // TEMP until "fields" hook available:
-  const status = data?.status ?? InvoiceNodeStatus.New;
-  const update = async (_: unknown) => true;
-  const onHold = false;
-  const lines: { totalCount: number; nodes: unknown[] } = {
-    totalCount: 0,
-    nodes: [],
-  };
+  const { mutateAsync } = useReturns.document.updateInboundReturn();
+
+  const {
+    data: { status, lines, onHold, id } = {
+      status: InvoiceNodeStatus.New,
+      lines: { totalCount: 0 },
+      onHold: false,
+    },
+  } = useReturns.document.inboundReturn();
```

Comment:
Trying this way instead of creating the `fields` hook just to get something working faster... but also I don't mind a little less indirection? Curious for thoughts, compared to slightly cleaner use with the fields hook?

Reply:
side note: there's a long running plan to refactor all of the hooks.
the 'fields' is particularly obscure and I'd like to change that one - I like what you've done here